portable/MSVC-MingW/port.c

```diff
@@ -246,8 +246,19 @@ StackType_t * pxPortInitialiseStack( StackType_t * pxTopOfStack,
                                                FALSE,  /* Start not signalled. */
                                                NULL ); /* No name. */
 
+
+#ifdef 	__x86_64__
```

Comment:
Why is this needed only on x64 and not on Win32?

Reply:
@aggarg
Thank you for pointing out. You are correct. Win32 is also needed.

I added `#pragma` only for x64 because I observed that MinGW32 does not report warning and I thought it is x64 unique.
But I found that MinGW32 version on my PC is too old and the latest MinGW32 reports same warning as MinGW64.
I modified `__x86_64__` to` __GNUC__`.
(I didn't delete `#ifdef` because MSVC does not recognize this pragma directive.)

Could you review modification?
I tested again all combinations of `TickType_t` width and compiler width on MinGW and MSVC.
No problem was found.

Reply:
Thank you for your contribution. Looks good!